apps/condo/domains/billing/gql.js

```diff
@@ -46,6 +47,7 @@ const BILLING_RECEIPT_ADMIN_FIELDS = `{ ${BILLING_RECEIPT_COMMON_FIELDS} file {
 
 const BillingReceipt = generateGqlQueries('BillingReceipt', BILLING_RECEIPT_FIELDS)
 const BillingReceiptAdmin = generateGqlQueries('BillingReceipt', BILLING_RECEIPT_ADMIN_FIELDS)
+const BillingReceiptBase = generateGqlQueries('BillingReceipt', `{ ${BASE_FIELDS} }`)
```

Comment:
May bi it’s better hardcode the id field here

Reply:
Probably I'm going to keep BASE_FIELDS, but put here all branches and so on, in order to use it like this:
generateGqlQueries('BillingReceipt', BASE_FIELDS)